src/components/Card/README.md

```diff
@@ -211,26 +211,6 @@ To cover all possible needs of your project, Card is available in colors from
     <Button label="Read more" priority="outline" color="note" />
   </CardFooter>
 </Card>
-<Card>
```

Comment:
I am just thinking about `Color Variants` generally. You removed `light` color, but it is still available through `null`, but this is not part of `Color Variants`. It is true that this is same for all components, but on the other hand, `null` is also color variants. This applies to Modal and other components as well.

This is just idea. If you think that it is valid, you can create an issue for that. If not, just let it be as it is. Anyway this is definitely scope of this issue.

Reply:
I see it the same as @mbohal described. What we need is this:

1. a neutral (light) variant for general purposes,
2. an option to highlight selected components with our feedback colors.

The key difference is that while the feedback variants define which border color or background color should be used, there is no such rule for the neutral appearance that would be applicable for all components. I mean I tried, but, with `Modal` colors (#468), it turned out it doesn't scale.